benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -102,16 +108,28 @@ export default class Overview extends React.Component {
 
   componentDidMount() {
     window.addEventListener("popstate", this.updateHiddenCols, false);
+    window.addEventListener("popstate", this.updateFilters, false);
```

Comment:
Not a really big deal, but I would suggest using a single handler `updateUrlParams` or something similar, that combines both `updateHiddenCols` and `updateFilters`. This would make the code a little bit more structured and less redundant, as we don't need to append two event handlers that each call `setState` and initialize a rerendering on their own.